beetsplug/ftintitle.py

```diff
@@ -82,6 +82,7 @@ def __init__(self):
         self.config.add({
             'auto': True,
             'drop': False,
+            'format': u'feat. {0}'
```

Comment:
A comma at the ends of this line helps with maintainability. (Tiny nitpick, I know. :flushed:)